src/accelerate/commands/launch.py

```diff
@@ -58,7 +58,7 @@ def launch_command_parser(subparsers=None):
         "--mixed_precision",
         default="no",
         type=str,
-        choices=["no", "fp16", "bf16"],
+        choices=PrecisionType,
```

Comment:
Here I think you might need to convert this in a list for `choices` to work the proper way.